libs/h3/src/main/java/org/opensearch/geospatial/h3/H3.java

```diff
@@ -35,6 +35,7 @@
  */
 public final class H3 {
 
+    public static int MIN_H3_RES = Constants.MIN_H3_RES;
```

Comment:
Any reason why we defining MIN_H3_RES here? Below code uses Constants.MIN_H3_RES.